src/Hooks.tsx

```diff
@@ -1,9 +1,9 @@
 import { useEffect, useState } from 'react'
 
-function useBrowserStorage<T>(
+export function useBrowserStorage<T>(
   key: string,
   initialValue: T,
-  type: 'session' | 'local',
+  type: 'session' | 'local' | 'cookie',
```

Comment:
Surely this shouldn't allow `'cookie'`? If `'cookie'` is selected, it will just use `localStorage`, considering the ternary on line 9..?